### PR TITLE
Adding delete api integration for drafts page and submit-for-review with updated figma.

### PR DESCRIPTION
**Pull request by Arti-Khillare1**

### CODE REVIEW

File: projects/lib-project/src/lib/lib-project.service.ts [hunk 1]
```diff
@@ -101,8 +101,17 @@ export class LibProjectService {
   }
 
   deleteProject(projectId:number|string){
-    return this.httpService.delete(this.Configuration.urlConFig.PROJECT_URLS.DELETE_PROJECT+projectId);
+    const config = {
+      url : `${this.Configuration.urlConFig.PROJECT_URLS.DELETE_PROJECT}/${projectId}`
+    };
+    return this.httpService.delete(config.url).pipe(
+      map((result : any) => {
+        console.log('Delete project successsfully', result);
```
Comment (ramkumar-ns-dev) [inline projects/lib-project/src/lib/lib-project.service.ts:109]: Aren't there any toaster after project deleted successfully?

File: projects/self-creation-portal/src/app/components/resource-holder/resource-holder.component.ts [hunk 2]
```diff
@@ -7,11 +7,14 @@ import { MatSidenavModule } from '@angular/material/sidenav';
...
 import { TranslateModule } from '@ngx-translate/core';
-import { ActivatedRoute } from '@angular/router';
+import { ActivatedRoute, Router } from '@angular/router';
 import { FormService } from '../../../../../lib-shared-modules/src/lib/services/form/form.service';
 import { ResourceService } from '../../services/resource-service/resource.service';
 import { SIDE_NAV_DATA } from '../../../../../lib-shared-modules/src/lib/constants/formConstant';
 import { CommonService } from '../../services/common-service/common.service';
+import { LibProjectService } from 'lib-project';
+import { MatDialog } from '@angular/material/dialog';
+import { DialogPopupComponent } from '../../../../../lib-shared-modules/src/public-api';
```
Comment (ramkumar-ns-dev) [inline projects/self-creation-portal/src/app/components/resource-holder/resource-holder.component.ts]: Why this import like this, not from library ?

Comment (ramkumar-ns-dev) [reply projects/self-creation-portal/src/app/components/resource-holder/resource-holder.component.ts]: Please change all other places as well.

**Review by ramkumar-pacewisdom — CHANGES_REQUESTED** [review]

File: projects/lib-project/src/lib/lib-project.service.ts (hunk 1 above)
Comment (Arti-Khillare1) [reply projects/lib-project/src/lib/lib-project.service.ts:109]: No there is no any toaster after project delete in figma I checked

**Review by ramkumar-pacewisdom — APPROVED** [review]

**Pull request merged**